tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -297,6 +297,174 @@ public function testCreateAttributes(array $data): array
+        ]);
+
+        $num_two_hundred = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/attributes/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => '200',
+            'size' => 256,
+            'required' => false,
+            'array' => false,
+        ]);
+
+        $num_million = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/attributes/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => '1000000',
+            'size' => 256,
+            'required' => false,
+            'array' => false,
+        ]);
+
+        $this->assertEquals(202, $num_one['headers']['status-code']);
+        $this->assertEquals($num_one['body']['key'], '1');
+        $this->assertEquals($num_one['body']['type'], 'string');
+        $this->assertEquals($num_one['body']['size'], 256);
+        $this->assertEquals($num_one['body']['required'], false);
+
+        $this->assertEquals(202, $num_two_hundred['headers']['status-code']);
+        $this->assertEquals($num_two_hundred['body']['key'], '200');
+        $this->assertEquals($num_two_hundred['body']['type'], 'string');
+        $this->assertEquals($num_two_hundred['body']['size'], 256);
+        $this->assertEquals($num_two_hundred['body']['required'], false);
+
+        $this->assertEquals(202, $num_million['headers']['status-code']);
+        $this->assertEquals($num_million['body']['key'], '1000000');
+        $this->assertEquals($num_million['body']['type'], 'string');
+        $this->assertEquals($num_million['body']['size'], 256);
+        $this->assertEquals($num_million['body']['required'], false);
+
+        // wait for database worker to create attributes
+        sleep(2);
+
+        $numeric_key_collection = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $numericKeyId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), []);
+
+        $this->assertIsArray($numeric_key_collection['body']['attributes']);
+        $this->assertCount(3, $numeric_key_collection['body']['attributes']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][0]['key'], $num_one['body']['key']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][1]['key'], $num_two_hundred['body']['key']);
+        $this->assertEquals($numeric_key_collection['body']['attributes'][2]['key'], $num_million['body']['key']);
```

Comment:
Let's remove these assertions to make the test easier to read, we only need to check the update works okay with the numeric keys